pandas/core/groupby/ops.py

```diff
@@ -655,15 +655,15 @@ def agg_series(self, obj, func):
             return self._aggregate_series_fast(obj, func)
         except AssertionError:
             raise
-        except ValueError as err:
+        except (ValueError, AttributeError) as err:
```

Comment:
what is the AttributeError in reference?

Reply:
See the issue https://github.com/pandas-dev/pandas/issues/29141. Now, since https://github.com/pandas-dev/pandas/pull/29100 is already merged, the AttributeError might not be needed anymore for our tests to pass.

Reply:
It only solves the fact that an AttributeError needs to be catched (I removed that in the last commit), it does not make the tests pass (as I mentioned yesterday)